frappe/contacts/doctype/address/address.js

```diff
@@ -57,5 +57,26 @@ frappe.ui.form.on("Address", {
 				}
 			}
 		]);
+	},
+	is_your_company_address: function(frm) {
+		if(frm.doc.is_your_company_address){
```

Comment:
The code dealing with "Company" shouldn't be in Frappe. First, we need to move the "Is Your Company Address" field to ERPNext, and then move this code to ERPNext as well.

Reply:
Ok closing this PR and will raise one in ERPNext with the customisations.